### ISSUE TITLE
Training on Trajnet++

### ISSUE THREAD

**Issue by Mirorrn** [vita-epfl/social-nce#5]
Hi I am about to try your code on the Trajnet++.

I am trying to get the FDE score of 1.14 .

Did you train on the whole (with cff) training dataset?
How many epochs?
And with which parameters?

If I reach your performance I will delete the submission if desired.

Thanks in advance
Many greetings

**Comment by YuejiangLIU** [comment]
Thanks for your interest in trying our method. 

To get the `FDE score of 1.14`, I simply followed the [instructions of Social LSTM baseline](https://github.com/vita-epfl/trajnetplusplusbaselines). 
- I did not use the cff dataset, which seems very noisy.
- I trained the baseline for 25 epochs.

After obtaining a competitive baseline model, I finetuned it using our S-NCE for 5 epochs.

I plan to release the code, once our submission to Trajnet++ is no longer ranked top.
